Laboratory/Rust/benchmarking/src/bebops/jazz.rs

```diff
@@ -491,3 +491,474 @@ impl<'raw> ::bebop::SubRecord<'raw> for Library<'raw> {
+            let len = ::bebop::read_len(&raw[i..])? + ::bebop::LEN_SIZE;
+            i += ::bebop::LEN_SIZE;
+
+            #[cfg(not(feature = "unchecked"))]
+            if len == 0 {
+                return Err(::bebop::DeserializeError::CorruptFrame);
+            }
+
+            if raw.len() < len {
+                return Err(::bebop::DeserializeError::MoreDataExpected(len - raw.len()));
+            }
+
+            let mut _title = None;
+            let mut _year = None;
+            let mut _performers = None;
+
+            #[cfg(not(feature = "unchecked"))]
+            let mut last = 0;
+
+            while i < len {
+                let di = raw[i];
+
+                #[cfg(not(feature = "unchecked"))]
+                if di != 0 {
+                    if di < last {
+                        return Err(::bebop::DeserializeError::CorruptFrame);
+                    }
+                    last = di;
+                }
+
+                i += 1;
+                match di {
+                    0 => {
+                        break;
+                    }
+                    1 => {
+                        #[cfg(not(feature = "unchecked"))]
+                        if _title.is_some() {
+                            return Err(::bebop::DeserializeError::DuplicateMessageField);
+                        }
+                        let (read, value) = ::bebop::SubRecord::_deserialize_chained(&raw[i..])?;
+                        i += read;
+                        _title = Some(value)
+                    }
+                    2 => {
+                        #[cfg(not(feature = "unchecked"))]
+                        if _year.is_some() {
+                            return Err(::bebop::DeserializeError::DuplicateMessageField);
+                        }
+                        let (read, value) = ::bebop::SubRecord::_deserialize_chained(&raw[i..])?;
+                        i += read;
+                        _year = Some(value)
+                    }
+                    3 => {
+                        #[cfg(not(feature = "unchecked"))]
+                        if _performers.is_some() {
+                            return Err(::bebop::DeserializeError::DuplicateMessageField);
+                        }
+                        let (read, value) = ::bebop::SubRecord::_deserialize_chained(&raw[i..])?;
+                        i += read;
+                        _performers = Some(value)
+                    }
+                    _ => {
+                        i = len;
+                        break;
+                    }
+                }
+            }
+
+            if i != len {
+                debug_assert!(i > len);
+                return Err(::bebop::DeserializeError::CorruptFrame)
+            }
+
+            Ok((i, Self {
+                title: _title,
+                year: _year,
+                performers: _performers,
+            }))
+        }
+    }
+
+    impl<'raw> ::bebop::Record<'raw> for  {}
```

Comment:
This empty `impl` for no type looks like something went wrong 🤔

Reply:
I am thinking I may have forgotten to update the jazz rust file since this would not even compile.